tracer/src/Datadog.Trace/Iast/SourceConverterDictionary.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="SourceConverterDictionary.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System.Collections.Generic;
+
+namespace Datadog.Trace.Iast;
+
+/// <summary>
+/// Convert <see cref="Datadog.Trace.Iast.Source"/> struct to a dictionary
+/// </summary>
+internal static class SourceConverterDictionary
+{
+    // When not redacted output is:
+    // { "origin": "http.request.parameter.name", "name": "name", "value": "value" }
+    //
+    // When redacted output is:
+    // { "origin": "http.request.parameter.name", "name": "name", "redacted": true }
+
+    public static Dictionary<string, object> ToDictionary(Source source, int maxValueLength)
```

Comment:
Not at all a priority, but it might be nice to make this an extension method so you can use it like this
```csharp
var dict = source.ToDictionary(maxLength);
```
instead of
```csharp
var dict = SourceConverterDictionary.ToDictionary(source, maxLength);
```

```suggestion
    public static Dictionary<string, object> ToDictionary(this Source source, int maxValueLength)
```